packages/app-desktop/gui/MainScreen/commands/showPrompt.ts

```diff
@@ -0,0 +1,34 @@
+import { CommandRuntime, CommandDeclaration, CommandContext } from '@joplin/lib/services/CommandService';
+
+export const declaration: CommandDeclaration = {
+	name: 'showPrompt',
+};
+
+interface PromptConfig {
+	label: string;
+	inputType?: 'dropdown' | 'datetime' | 'tags' | 'text';
```

Comment:
Your code is correct, but TypeScript has various issues with fields defined as string options like this. So instead it's better to create an enum type that has all the possible values.

So `inputType?: PromptInputType`

And then:

```typescript
enum PromptInputType {
    Dropdown = 'dropdown',
    // etc.
}
```

Reply:
Ah okay. Done.